packages/xo-web/src/common/xo/index.js

```diff
@@ -774,13 +774,14 @@ export const installHostPatch = (host, { uuid }) =>
     patches: [uuid],
   })::tap(() => subscribeHostMissingPatches.forceRefresh(host))
 
-export const installAllHostPatches = host =>
-  _call('pool.installPatches', { hosts: [resolveId(host)] })::tap(() =>
-    subscribeHostMissingPatches.forceRefresh(host)
-  )
+export const installPatches = (patches, pool) =>
+  _call('pool.installPatches', {
+    pool: resolveId(pool),
+    patches: resolveIds(patches),
+  })::tap(() => subscribeHostMissingPatches.forceRefresh())
 
 import InstallPoolPatchesModalBody from './install-pool-patches-modal' // eslint-disable-line import/first
-export const installAllPatchesOnPool = pool => {
+export const installAllPatchesOnPool = ({ pool }) => {
```

Comment:
This change breaks [this](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/common/hosts-patches-table.js#L112-L119).